siliconcompiler/core.py

```diff
@@ -2690,6 +2691,7 @@ def dashboard(self, wait=True, port=None, graph_chips=None, type=DashboardType.W
             self._dash = WebDashboard(self, port=port, graph_chips=graph_chips)
         elif type == DashboardType.CLI:
             self._dash = CliDashboard(self)
+            atexit.register(self._dash.stop)
```

Comment:
could this be in the initializer for the CLIDashboard? that way it would get handled automatically and less prove to be forgotten?

Reply:
Moving it into BoardSingleton seems to work fine. Thanks.